eda/eda_election_turnout.Rmd

```diff
@@ -0,0 +1,172 @@
+---
```

Comment:
I needed to create a data/raw folder within the eda folder to knit this file. Is this expected behaviour or should it reference a data/raw folder outside of the eda folder?

Reply:
Correct, it requires the two CSVs to live in `data/raw`. I'm assuming that will come in the data-downloading PR?